frame/treasury/src/tests.rs

```diff
@@ -119,14 +110,36 @@ impl Config for Test {
 	type SpendFunds = ();
 }
 
+type UncheckedExtrinsic = frame_system::mocking::MockUncheckedExtrinsic<Test>;
+type Block = frame_system::mocking::MockBlock<Test>;
+
+frame_support::construct_runtime!(
+	pub enum Test where
+		Block = Block,
+		NodeBlock = Block,
+		UncheckedExtrinsic = UncheckedExtrinsic,
+	{
+		System: frame_system::{Pallet, Call, Storage, Config, Event<T>},
+		Balances: pallet_balances::{Pallet, Call, Storage, Config<T>, Event<T>},
+		Treasury: treasury::{Pallet, Call, Storage, Config<T>, Event<T>},
+	}
+);
```

Comment:
why did you moved it ?

Reply:
In some refernce test file saw that `construct_runtime!()` called after 
defining all trait configs. Revert back the position.